### ISSUE TITLE
Can't commit no more :(

### ISSUE THREAD

**Issue by hedefalk** [sbt/sbt-git#77]
```
> git commit -m "gulp html test"
[error] Expected whitespace character
[error] Expected non-double-quote-space character
[error] git commit -m "gulp html test"
[error]
```

**Comment by bchazalet** [comment]
Ouch.. that's bad. It must be a bug introduced by the branch auto-completion feature, I will have a look.

**Comment by jsuereth** [comment]
Fixed in 0.8.3
